doc/release-notes.md

```diff
@@ -91,6 +91,10 @@ Low-level RPC changes
 - The wallet RPC `getreceivedbyaddress` will return an error if called with an address not in the wallet.
 
 
+- `listwallets` now returns two arrays of wallet names, one listing the `loaded` wallets
```

Comment:
I don't think it's a big deal to break backwards compatibility here, but I also don't see any benefit in doing so. I'd think that a better approach to overloading the `listwallets` function would be to add a new `listwalletdir` function that would return a list of wallet files inside `-walletdir`. Advantages to this approach:

1) Backwards compatible
2) Better performance for listwallets. It could continue to just return the loaded wallets from memory instead of having now having to do disk io.
3) More descriptive function name. `listwalletdir` naming is consistent with recently added `walletdir` parameter, and tells you what the function actually does.